edward/inferences/sgld.py

```diff
@@ -97,25 +97,26 @@ def log_joint(self, z_sample):
       Latent variable keys to samples.
     """
     if self.model_wrapper is None:
+      scope = 'inference_' + str(id(self))
```

Comment:
yeah, i'm not sure why they have different scopes in the first place. apparently i only did this for Monte Carlo methods and not any variational inference methods. i would not think not, but maybe my past self knows more.